src/schemas/accounts.py

```
@@ -0,0 +1,71 @@
+
+class BaseEmailPasswordSchema(BaseModel):
+    email: EmailStr
+    password: str
+
+    model_config = {
+        "from_attributes": True
+    }
+
+    @field_validator("email")
+    @classmethod
+    def validate_email(cls, value):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)